<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>截图控制</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>截图间隔（分钟）:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="intervalSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>选择屏幕:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="screenCombo"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="startCapture">
         <property name="text">
          <string>开始自动截图</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="stopCapture">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止自动截图</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QPushButton" name="manualCapture">
         <property name="text">
          <string>立即截图</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>文件保存</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>保存路径:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="folderPath"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="selectFolder">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="refreshList">
           <property name="text">
            <string>刷新列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteSelected">
           <property name="text">
            <string>删除选中</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteOldFiles">
           <property name="text">
            <string>删除旧文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="fileTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>就绪</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="countdownLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="storageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>